### PR TITLE
Update DYCOMS subsidence parameter to reflect 14400s offline simulations

### PR DESCRIPTION
**Pull request by smarras79**
# Description

 experiments/AtmosLES/dycoms.jl:
 Modified dycoms.jl to have the correct parameters that allow it to run to completion if timeend is set to 14400. 
   * D_subsidence = 3.75e-6
   * maxiter = 1
   * tolerance = FT(50)
   * Courant_number = 1.8
These are the parameters that allowed me to run dycoms to completion off-line. I used these parameters to produce the first results plotted in the dycoms paper draft (overleaf) and in the slack #dycoms channel.

# For review by CLIMA developers

- [x] There are no open pull requests for this already
- [x] CLIMA developers with relevant expertise have been assigned to review this submission
- [x] The code conforms to the [style guidelines](https://climate-machine.github.io/CLIMA/latest/CodingConventions.html) and has consistent naming conventions
- [x] This code does what it is technically intended to do (all numerics make sense physically and/or computationally)

### CODE REVIEW

**Comment by smarras79** [comment]
bors try

File: experiments/AtmosLES/dycoms.jl [hunk 1]
```diff
@@ -242,7 +242,7 @@ function init_dycoms!(bl, state, aux, (x,y,z), t)
     z = altitude(bl.orientation, aux)
 
     # These constants are those used by Stevens et al. (2005)
```
Comment (akshaysridhar) [inline experiments/AtmosLES/dycoms.jl]: This value 8.5e-3 is not the same as that defined in the paper ? Would suggest removing this comment in that case.

File: experiments/AtmosLES/dycoms.jl [hunk 2]
```diff
@@ -306,7 +306,7 @@ function config_dycoms(FT, N, resolution, xmax, ymax, zmax)
     α_z           = FT(1)
     z_i           = FT(840)
     ρ_i           = FT(1.13)
-    D_subsidence  = FT(0) # 0 for stable testing, 3.75e-6 in practice
+    D_subsidence  = FT(3.75e-6) # 0 for stable testing, 3.75e-6 in practice
```
Comment (akshaysridhar) [inline experiments/AtmosLES/dycoms.jl]: The intent here is to modify this to be consistent with your stable `offline` simulations at `timeend = 14400s` , which means this comment about stable testing vs practice can be removed.

File: experiments/AtmosLES/dycoms.jl [hunk 3]
```diff
@@ -376,9 +376,9 @@ function main()
     timeend = FT(100)
```
Comment (akshaysridhar) [inline experiments/AtmosLES/dycoms.jl]: @simonbyrne @smarras79 What is the longest simulation `timeend` we can afford to link against the  current `CI` pipeline ?

File: experiments/AtmosLES/dycoms.jl (hunk 1 above)
Comment (akshaysridhar) [inline experiments/AtmosLES/dycoms.jl]:
```suggestion
```

File: experiments/AtmosLES/dycoms.jl (hunk 2 above)
Comment (akshaysridhar) [reply experiments/AtmosLES/dycoms.jl]:
```suggestion
    D_subsidence  = FT(3.75e-6) 
```

File: experiments/AtmosLES/dycoms.jl (hunk 3 above)
Comment (akshaysridhar) [inline experiments/AtmosLES/dycoms.jl]:
```suggestion
    timeend = FT(3600)
```

Comment (kpamnany) [reply experiments/AtmosLES/dycoms.jl]: It's completely parallel so there's no time-to-completion cost. The current budget per run is 60 minutes.

**Comment by akshaysridhar** [comment]
bors try

**Comment by akshaysridhar** [comment]
bors try

File: experiments/AtmosLES/dycoms.jl [hunk 4]
```diff
@@ -341,7 +340,7 @@ function config_dycoms(FT, N, resolution, xmax, ymax, zmax)
     model = AtmosModel{FT}(AtmosLESConfigType;
                            ref_state=ref_state,
                           turbulence=SmagorinskyLilly{FT}(C_smag),
-                            moisture=EquilMoist{FT}(;maxiter=5),
+                            moisture=EquilMoist{FT}(maxiter=1, tolerance=FT(50)),
```
Comment (charleskawczynski) [inline experiments/AtmosLES/dycoms.jl]:
```suggestion
                            moisture=EquilMoist{FT}(;maxiter=1, tolerance=FT(50)),
```

File: experiments/AtmosLES/dycoms.jl [hunk 5]
```diff
@@ -376,9 +375,9 @@ function main()
     timeend = FT(100)
 
     driver_config = config_dycoms(FT, N, resolution, xmax, ymax, zmax)
-    solver_config = CLIMA.setup_solver(t0, timeend, driver_config, init_on_cpu=true)
+    solver_config = CLIMA.setup_solver(t0, timeend, driver_config, Courant_number=1.8, init_on_cpu=true)
```
Comment (charleskawczynski) [inline experiments/AtmosLES/dycoms.jl]:
```suggestion
    solver_config = CLIMA.setup_solver(t0, timeend, driver_config; Courant_number=1.8, init_on_cpu=true)
```
Assuming all kwargs come after `driver_config`

Comment (akshaysridhar) [reply experiments/AtmosLES/dycoms.jl]: Fixed :D

File: experiments/AtmosLES/dycoms.jl (hunk 4 above)
Comment (akshaysridhar) [reply experiments/AtmosLES/dycoms.jl]: fixed, thanks

**Comment by akshaysridhar** [comment]
bors try

**Pull request merged**